day_two_guide/topics/controller_role_verification.adoc

```diff
@@ -0,0 +1,43 @@
+////
+Controller Role Verification
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_health_checks.adoc
+////
+
+The {product-title} controller service,
+`atomic-openshift-master-controllers.service`, is available across all master
+hosts. The service runs in active/passive mode, meaning it should only be
+running on one master at any time.
+
+The {product-title} controllers execute a procedure to choose which host runs
+the service. The current running value is stored in the `etcd` cluster within the `/openshift.io/leases/controllers` key.
```

Comment:
OpenShift used the new mechanism to perform controller election since 3.7:

Changes is here: https://github.com/openshift/openshift-ansible/pull/4832

Reply:
This was written before 3.7 was released.